vendor.conf

```diff
@@ -63,34 +63,36 @@ github.com/cilium/ebpf                              4032b1d8aae306b7bb94a2a11002
 github.com/davecgh/go-spew                          v1.1.1
 github.com/docker/spdystream                        449fdfce4d962303d702fec724ef0ad181c92528
 github.com/emicklei/go-restful                      v2.9.5
+github.com/go-logr/logr                             v0.2.0
 github.com/google/gofuzz                            v1.1.0
-github.com/json-iterator/go                         v1.1.8
+github.com/json-iterator/go                         v1.1.9
 github.com/modern-go/concurrent                     1.0.3
 github.com/modern-go/reflect2                       v1.0.1
 github.com/pmezard/go-difflib                       v1.0.0
 github.com/seccomp/libseccomp-golang                v0.9.1
 github.com/stretchr/testify                         v1.4.0
 golang.org/x/crypto                                 bac4c82f69751a6dd76e702d54b3ceb88adab236
-golang.org/x/oauth2                                 0f29369cfe4552d0e4bcddc57cc75f4d7e672a33
-golang.org/x/time                                   9d24e82272b4f38b78bc8cff74fa936d31ccd8ef
+golang.org/x/oauth2                                 858c2ad4c8b6c5d10852cb89079f6ca1c7309787
+golang.org/x/time                                   555d28b269f0569763d25dbe1a237ae74c6bcc82
```

Comment:
diff for these deps: 

- https://github.com/golang/oauth2/compare/0f29369cfe4552d0e4bcddc57cc75f4d7e672a33...858c2ad4c8b6c5d10852cb89079f6ca1c7309787
- https://github.com/golang/time/compare/9d24e82272b4f38b78bc8cff74fa936d31ccd8ef...555d28b269f0569763d25dbe1a237ae74c6bcc82

Reply:
no harm in updating these right?

Reply:
looks fine; was checking if the commits were newer (not older) than the existing ones, and thought I'd post the diff urls for others to review